transports/sftp/src/main/java/org/mule/transport/sftp/SftpMessageReceiver.java

```diff
@@ -332,4 +350,24 @@ protected void doDispose()
     {
         // no op
     }
+
+    private class SFTPMessageFactory extends DefaultMuleMessageFactory
```

Comment:
Are you sure DefaultMuleMessageFactory was being used and not AbstractMuleMessageFactory?